pkg/batch/cmd/server/server.go

```diff
@@ -560,6 +530,61 @@ func (s *server) createMySQLClient(
 	)
 }
 
+func (s *server) startMultiPubSub(
+	ctx context.Context,
+	processors *processor.Processors,
+	logger *zap.Logger,
+) *subscriber.MultiSubscriber {
+	bytes, err := os.ReadFile(*s.subscriberConfig)
+	if err != nil {
+		logger.Fatal("subscriber: failed to read subscriber config",
+			zap.Error(err),
+		)
+		return nil
+	}
+	var configMap map[string]subscriber.Configuration
+	if err := json.Unmarshal(bytes, &configMap); err != nil {
+		logger.Fatal("subscriber: failed to unmarshal subscriber config",
+			zap.Error(err),
+		)
+		return nil
+	}
+	multiSubscriber := subscriber.NewMultiSubscriber(
+		subscriber.WithLogger(logger),
+	)
+	for name, config := range configMap {
+		p, err := processors.GetProcessorByName(name)
+		if err != nil {
+			logger.Fatal("subscriber: processor not found",
+				zap.String("name", name),
+				zap.Error(err),
+			)
+			return nil
```

Comment:
It would be safer if we return an error here. Otherwise, we won't notice that the subscription is not working when the name doesn't match the processor name.